pkg/server/nfs/nfs_server.go

```diff
@@ -133,3 +145,25 @@ func setRlimitNOFILE(logger logrus.FieldLogger) error {
 	logger.Infof("ending RLIMIT_NOFILE rlimit.Cur %d, rlimit.Max %d", rlimit.Cur, rlimit.Max)
 	return nil
 }
+
+func SetConfigParams(config string) string {
```

Comment:
Suggest renaming to `updateGaneshaConfigFile`. 

note: You don't need to export this function w/ the first letter capitalized.